diff --git a/third_party/abseil/absl/crc/internal/crc_memcpy_x86_arm_combined.cc b/third_party/abseil/absl/crc/internal/crc_memcpy_x86_arm_combined.cc
--- a/third_party/abseil/absl/crc/internal/crc_memcpy_x86_arm_combined.cc
+++ b/third_party/abseil/absl/crc/internal/crc_memcpy_x86_arm_combined.cc
@@ -161,7 +161,7 @@ crc32c_t AcceleratedCrcMemcpyEngine<vec_regions, int_regions>::Compute(
     crc32c_t initial_crc) const {
   constexpr std::size_t kRegions = vec_regions + int_regions;
   static_assert(kRegions > 0, "Must specify at least one region.");
-  constexpr uint32_t kCrcDataXor = uint32_t{0xffffffff};
+  constexpr uint32_t kCrcDataXor = 0xffffffffu;
   constexpr std::size_t kBlockSize = sizeof(V128);
   constexpr std::size_t kCopyRoundSize = kRegions * kBlockSize;

